packages/ember-metal/lib/map.js

```diff
@@ -63,7 +63,7 @@ class OrderedSet {
   /**
     @method create
     @static
-    @return {Ember.OrderedSet}
+    @return {OrderedSet}
```

Comment:
According to https://github.com/ember-cli/ember-rfc176-data/blob/master/mappings.json its still `Ember.OrderedSet`